src/store/enhancedImageBuilderApi.ts

```diff
@@ -20,6 +20,31 @@ const enhancedApi = imageBuilderApi.enhanceEndpoints({
         return [{ type: 'Clone', id: arg.composeId }];
       },
     },
+    createBlueprint: {
+      onQueryStarted: async (_, { dispatch, queryFulfilled }) => {
+        queryFulfilled
+          .then(() => {
+            // Typescript is unaware of tag types being defined concurrently in enhanceEndpoints()
+            // @ts-expect-error
```

Comment:
linter in the iqe pipeline is saying this is unused (meaning ts would not error?)